### ISSUE TITLE
Pre-processing after re-basecalling requires full path

### ISSUE THREAD

**Issue by lkwhite** [rnabioco/ont-pipeline#2]
For reasons that remain opaque to me, 00_preprocess.sh fails to complete after re-basecalling with Guppy _unless_ you specify the full path to the fast5 and fastq files. Unless I'm missing something, if you specify a correct _relative_ path:

- multi_to_single_fast5 will convert zero files
- tombo preprocess annotate_raw_with_fastqs will produce a FileNotFoundError

**Comment by jayhesselberth** [comment]
Seems like a an easy fix, just provide a full path?

**Comment by lkwhite** [comment]
Yeah, I didn't put this here because I need you to fix it, just to remind
myself I'll need to fix it eventually.
